src/components/SelectComponents/Autosuggest/SuggestionTag/__tests__/SuggestionTag.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { SuggestionTag } from '../SuggestionTag';
+
+describe('SuggestionTag', () => {
+    it('should render nothing if no children are provided', () => {
+        const view = render(<SuggestionTag />);
+
+        expect(view.baseElement).toMatchSnapshot();
```

Comment:
Why here is using baseElement and not container? In the rest of the places we are using view.container.

Reply:
For container it returns 
![image](https://github.com/textkernel/oneui/assets/104063210/8e7b6383-86d0-4f5a-9630-517046ddebd8)
For baseElement 
![image](https://github.com/textkernel/oneui/assets/104063210/26fac668-8cfc-4aa5-b6ad-954aa4df611a)

About baseElement we have some cases where we use it. In this case i think we can use both

Reply:
In the cases that we are using `baseElement` instead of the `container` what are the reasons behind it?
Because accordingly to the guidelines present on TESTING_LIBRARY_MIGRATION.md we should use `container`.

Reply:
Okay, i've changed it